test/nodejs/logging/logging.js

```diff
@@ -181,9 +198,17 @@ describe('loggingV2', function() {
         }
         expect(found).to.be.true;
         return client.deleteLogMetric({metricName: metricName});
+        // catch(done) should be here to catch all failures until this line.
+        // The next API call is expected to fail.
+      }).catch(done).then(function() {
+        return client.getLogMetric({metricName: metricName});
       }).then(function() {
+        done(new Error(
+          'The last getLogMetric should not succeed because it is already deleted'));
+      }).catch(function(err) {
```

Comment:
I would add a check here to make sure that this is the correct error thrown from `client.getLogMetric({metricName: metricName});` after the metric has been deleted. If it isn't checked, a failure in the previous `.then` statements would fail silently.